pkg/ui/src/views/cluster/containers/timescale/index.tsx

```diff
@@ -49,7 +49,12 @@ interface TimeScaleDropdownProps extends RouteComponentProps {
 }
 
 export const getTimeLabel = (currentWindow?: timewindow.TimeWindow, windowSize?: moment.Duration) => {
-  const time = windowSize ? windowSize : moment.duration(moment(currentWindow.end).diff(currentWindow.start));
+  let time = moment.duration(10, "minutes");
+  if (windowSize) {
+    time = windowSize;
+  } else if (currentWindow) {
+    time = moment.duration(moment(currentWindow.end).diff(currentWindow.start));
+  }
```

Comment:
I don't care for this change as it introduces mutability. The code in this case is fine, but it set a bad precedent and time is now able to mutable later in the code in unexpected ways. I assume this change was made for readibiltiy, and having it on multiple lines is indeed more readable, but I would keep the ternary statement and spread that across multiple lines instead.

```suggestion
  const time = windowSize ? 
  windowSize : 
  (currentWindow) ?
  moment.duration(moment(currentWindow.end).diff(currentWindow.start)) :
  moment.duration(10, "minutes");
```

The formatting here is perhaps up for debate (I would prefer to defer to [ESLint](https://eslint.org/docs/rules/multiline-ternary)), but I would like to keep the code immutable unless strictly necessary.